lmdeploy/lite/utils/load.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+import torch
+from accelerate import infer_auto_device_map, init_empty_weights
+from transformers import AutoConfig, AutoModelForCausalLM
+
+from lmdeploy.lite.utils import collect_target_modules
+from lmdeploy.pytorch.model import LoadWoInit
+
+LAYER_TYPE_MAP = {
+    'InternLMForCausalLM': 'InternLMDecoderLayer',
+    'QWenLMHeadModel': 'QWenBlock',
+    'BaiChuanForCausalLM': 'DecoderLayer',
```

Comment:
BaiChuanForCausalLM supports both baichuan 1 and 2?

Reply:
My bad! Fixed.